index.html

```diff
@@ -0,0 +1,79 @@
+        <p><i>Data source: American Community Survey 2022 5-year estimates</i></p>
+      </article>
+      
+      <article class="slide" id="3-CB-slide">
+        <h2>Factors affecting food insecurity: Rent Cost-Burden</h2>
+        <p>Another huge contributor to food insecurity is housing cost-burden. Households are cost-burdened if housing costs take up at least 30% of their entire household income. This reduces  the disposable income remaining to spend on food. With such a large percentage of their income going into paying for rent, these households are sometimes forced to sacrifice food on the table for a roof over their heads. Cost-burdened households generally correlate with poverty levels, with higher percentages in north and west Philadelphia.</p>
+        <p><i>Data source: American Community Survey 2022 5-year estimates</i></p>
+      </article>
+
+      <article class="slide" id="4-Access-slide">
+        <h2>Factors affecting food insecurity: Access to Healthy Foods</h2>
+        <p>Another factor that contributes to food insecurity is the geographical accessibility to cost-friendly nutritional food. ‘Food deserts’ are geographic areas where there is limited access to affordable and healthful food options. This map shows the relative access within a half-mile walking distance to high-produce supply stores, which generally offer more healthy and fresh produce. While there is high access to such stores in center city, there are less stores in the far north and south regions. </p>
+        <p><i>Data source: Neighborhood Food Retail 2019</i></p>
+      </article>
+
+      <article class="slide" id="5-FS-slide">
+        <h2>Supplemental Nutrition Assistance Program (SNAP) in Philadelphia</h2>
+        <p>To tackle this issue, the government in the US provides households with food benefits under the Supplemental Nutrition Assistance Program, which support low-income households in affording groceries. The percentage of households in Philadelphia receiving SNAP food benefits corresponds greatly to relative poverty levels. However, critics have pointed out that the program provides insufficient benefits and has failed to offer households nutritional food, with SNAP beneficiaries having poorer diets than non-SNAP households.</p>
+        <p><i>Data source: American Community Survey 2022 5-year estimates.</i></p>
+      </article>
+
+      <article class="slide" id="6-ShareFood-slide">
+        <h2>Taking Action: Share Food Program's North Philadelphia Warehouse </h2>
```

Comment:
**praise:** I like the conceptual organization of your slides into sections (i.e. _Factors affecting food insecurity_, _Taking Action_)